src/modules/comments/comments-app.js

```diff
@@ -0,0 +1,164 @@
+import CarsApi from '../home/cars-api';
+import CommentData from './commentData';
+import Api from './createComments';
+import commentCounter from './comment-counter';
+
+const carApi = new CarsApi();
+
+const modalContainer = document.querySelector('#modal-container');
+
+const renderComment = (dataItem) => `<li class="text-lg">
+        <span>${dataItem.creation_date}</span> -
+        <span class="text-slate-600 mx-2">${dataItem.username} : </span>
+        <span>${dataItem.comment}</span>
+      </li>`;
+
+const displayCommentCounter = (data) => {
+  const numberOfComments = commentCounter(data);
+  const commentCounterElement = document.querySelector(
+    '#modal .comment-counter-header',
+  );
+
+  commentCounterElement.textContent = `(${numberOfComments})`;
+};
+
+const displayComment = (data) => {
+  const CommentContainer = document.querySelector('#modal .comment-container');
+  if (data.length > 0) {
+    let containerString = '';
+    data.forEach((dataItem) => {
+      containerString += `${renderComment(dataItem)} \n`;
+    });
+
+    CommentContainer.innerHTML = containerString;
+  } else {
+    CommentContainer.innerHTML = '';
+  }
+};
+
+export const createComment = (event) => {
+  event.preventDefault();
+  const api = new Api();
+  const form = document.querySelector('.form');
+  const comBtn = document.querySelector('.form button');
+  const id = Number(comBtn.id.split('-')[2]);
+  const usernameInput = document.querySelector('.form .input');
+  const commentInput = document.querySelector(
+    '#modal-container .modal-items form textarea',
+  );
+
+  const username = usernameInput.value;
+  const comment = commentInput.value;
+  const commentObj = new CommentData(username, comment, id);
+
+  api
+    .addComment(commentObj)
+    .then(() => api.getComment(id))
+    .then((data) => {
+      displayComment(data);
+      displayCommentCounter(data);
+    });
+  form.reset();
+};
+
+export async function showCommentModal(e) {
```

Comment:
- [ ] according to the requirements the Project should use ES6, which means you should avoid usage of the function keyword, please use arrow functions instead.